mimic/rest/heat_api.py

```diff
@@ -75,6 +75,9 @@ class HeatRegion(object):
     Rest endpoints for mocked Heat API.
     """
 
+    HEAT_URL = '/v1/<string:tenant_id>'
```

Comment:
Non-blocking: Would you consider renaming this to `TENANT_URL`? I would consider `HEAT_URL` to be `.../stacks` which you've already correctly named as `STACK_URL` :)